elastic/kibana
# [Cases] Refactor Cases List Filters

Meta https://github.com/elastic/kibana/issues/167651

### Description
This PR refactors the filters to be multi select

<img width="829" alt="Screenshot" src="https://github.com/elastic/kibana/assets/17549662/5b0efe3f-f4a0-423e-9d64-0dfa50e74ce6">

### QA
- Filters work as multi select now
- 'All' has been removed as option for status and severity. Check that old urls, local storage values do not break anything
- Review the cases list in the modal when adding a timeline to a case
